src/coreclr/jit/morph.cpp

```diff
@@ -10678,6 +10678,63 @@ GenTree* Compiler::fgOptimizeHWIntrinsic(GenTreeHWIntrinsic* node)
             INDEBUG(node->gtDebugFlags |= GTF_DEBUG_NODE_MORPHED);
             return node;
         }
+#if defined(TARGET_ARM64)
```

Comment:
Is there an issue tracking mirroring of the other mask related handling that morph handles for xarch?

In particular, the recognition of `And(MaskToVector(x), MaskToVector(y))` and converting it to `AndMask(x, y)` (as well as similar transforms for other trivial operations where the operation is identical).

Reply:
https://github.com/dotnet/runtime/issues/101933